builddefs/common_features.mk

```diff
@@ -183,7 +183,7 @@ else
     ifeq ($(PLATFORM),AVR)
       # Automatically provided by avr-libc, nothing required
     else ifeq ($(PLATFORM),CHIBIOS)
-      ifneq ($(filter STM32F3xx_% STM32F1xx_% %_STM32F401xC %_STM32F401xE %_STM32F405xG %_STM32F411xE %_STM32F072xB %_STM32F042x6 %_GD32VF103xB %_GD32VF103x8, $(MCU_SERIES)_$(MCU_LDSCRIPT)),)
+      ifneq ($(filter STM32F3xx_% STM32F1xx_% STM32F4xx_% %_STM32F401xC %_STM32F401xE %_STM32F405xG %_STM32F411xE %_STM32F072xB %_STM32F042x6 %_GD32VF103xB %_GD32VF103x8, $(MCU_SERIES)_$(MCU_LDSCRIPT)),)
```

Comment:
I missed this changes with my last rebase.

Also i find see other checks for mcus below as redundant, since all of these mcus define MCU_SERIES as STM32F4xx
```_STM32F401xC %_STM32F401xE %_STM32F405xG %_STM32F411xE```

Shall i change remove the above checks and keep only `STM32F4xx_%` ?
Or should i use my ld script value to enable EEPROM_DRIVER

I may be missing something here. Please advise